### PR TITLE
Create Godot project

### PR DESCRIPTION
**Pull request by xix-xeaon**
We need a project for everyone to work on.

### CODE REVIEW

**Comment by ExclusiveKnight** [comment]
What version of Godot is used to create the project?

**Review by Tobeast1 — APPROVED** [review]
Very good hopefully not a virus

**Review by lawin666 — APPROVED** [review]

File: .gitignore
Comment (ExclusiveKnight) [inline .gitignore:1]: Many of these ignores are removed, are they necessary? I know the **.import/** folder is automatically created when you add an image to the project.

Comment (xix-xeaon) [reply .gitignore:1]: It's not necessary for us.

Comment (ExclusiveKnight) [reply .gitignore:1]: Okay. Approved.

**Review by ExclusiveKnight — APPROVED** [review]

**Pull request merged**